test/test_dimes.py

```diff
@@ -75,12 +75,13 @@ def test_multi_plot():
     )
     # Time series and axis will get name from dimensionality
     plot.add_time_series(
-        TimeSeriesData([x for x in plot.time_values], native_units="ft", display_units="cm")
+        TimeSeriesData([x for x in plot.time_values], native_units="ft", display_units="cm"),
+        subplot_number=2
```

Comment:
We still want to test a plot getting added to a default subplot. Instead, we should create another timeseries for the third subplot.

Reply:
We should update the comment to indicate this too.